plugins/cortex_v2/icon_cortex_v2/util/util.py

```diff
@@ -0,0 +1,39 @@
+from typing import List, Dict
+
+
+def filter_analyzers(analyzers: List[Dict]):
+    return [filter_analyzer(analyzer) for analyzer in analyzers]
+
+
+def filter_analyzer(analyzer: Dict):
+    return {
+        k: v
+        for k, v in analyzer.items()
+        if k in {"id", "version", "dataTypeList", "name", "description", "license", "author", "url"}
+    }
```

Comment:
```suggestion
from typing import List, Dict

RELEVANT_ANALYZER_KEYS = {"id", "version", "dataTypeList", "name", "description", "license", "author", "url"}

def filter_analyzers(analyzers: List[Dict]):
    return [filter_analyzer(analyzer) for analyzer in analyzers]


def filter_analyzer(analyzer: Dict):
    return {
        k: v
        for k, v in analyzer.items()
        if k in RELEVANT_ANALYZER_KEYS
    }
```
Just a little bit more readable